bfe_basic/common.go

```diff
@@ -47,7 +47,7 @@ const (
 	GlobalProduct = "global"
 )
 
-// Hop-by-hop headers. These are removed when sent to the backend.
+// HopHeaders Hop-by-hop headers. These are removed when sent to the backend.
```

Comment:
// HopHeaders are Hop-by-hop headers. These are removed when sent to the backend.